SwiftJym/Exercise.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Excercise.swift
+//  SwiftJym
+//
+//  Created by Andrea Rota on 11/7/23.
+//
+
+import Foundation
+
+struct Exercise: Identifiable {
+    
+    let id = UUID()
+    let name: String
+    let image: String // TODO: check if an array is needed or 1:1
```

Comment:
I've seen that the image's name end with a 1 so wasn't sure that only one string for exercise / image is enough